### PR TITLE
fix: bitbucket is repo provider

### PR DESCRIPTION
**Pull request by chohmann**

### CODE REVIEW

File: docs/external-services.md [hunk 1]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+5. *billing*: someone who only performs billing related actions
+
+*Stoplight Permissions*
+
+1. *ci:read*: allows users to view project ci tokens and the instructions on how to use them during the analyze process
+2. *ci:write*: allows users to re-generate project ci tokens
+3. *org:read*: allows users to view content belonging to the org
+4. *org:write*: allows users to change org settings
+5. *org:delete*: allows users to delete an org (and all of its projects) from Stoplight
+6. *org:push*: allows users edit content belonging to org
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
6. *org:push*: allows users to edit content belonging to org
```

File: docs/external-services.md [hunk 2]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+7. *billing:write*: allows users modify billing information
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
7. *billing:write*: allows users to modify billing information
```

File: docs/external-services.md [hunk 3]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+8. *billing:read*: alows users to view billing information
+
+> Permissions will never be able to be edited by Stoplight users. If we release a new feature/product that requires a new permission, we will write a DB migration that will add this permission. We will then mention in release notes to customers what the permission enables and that they can add it to roles to give users access to the new feature.
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
> Permissions will never be able to be edited by Stoplight users. If we release a new feature/product that requires new permissions, we will write a DB migration that will add this permission. We will then mention in release notes to customers what the permission enables and that they can add it to roles to give users access to the new feature.
```

File: docs/external-services.md [hunk 4]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
 7. Jane, a developer on Bob's team, creates a Stoplight account by logging in with Github.
 8. Step 3 repeats itself, but for Jane. This time, there ARE orgs and projects inside of Stoplight (one org, and three projects to be exact). In fact, Jane is a member of the organization that Bob already added to Stoplight! Thus, when we ask Github for the list of orgs and projects Jane has access to, we are able to match the external github org with the org in Stoplight, and add a row in our DB indicating Jane is a reader in this org.
 
+**Stoplight Permissioning**
+
+The following roles, permissions, and mappings will be created by default. For the first phase, Stoplight users will not be able to add, modify, delete these within Stoplight. In a future phase, we will allow Stoplight users to maintain custom role setups.
```
Comment (brianmrock) [inline docs/external-services.md]: removed a couple instances of furture present tense ... technical writing should attempt to be in present tense in most cases.  

```suggestion
The following roles, permissions, and mappings are created by default. For the first phase, Stoplight users can add, modify, delete these within Stoplight. In a future phase, we will enable Stoplight users to maintain custom role setups.
```

Comment (chohmann) [reply docs/external-services.md]: is this true? We won't allow users to edit roles/permissions in the first phase

File: docs/external-services.md [hunk 5]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+
+1. admin --> ci:read, ci:write, org:read, org:write, org:delete, org:push, billing:write, billing:read
+2. owner --> org:read, org:write, org:delete, org:push 
+3. writer --> org:read, org:push
+4. reader --> org:read
+5. billing --> billing:write, billing:read
+
+**Stoplight to External Serice Role Mappings**
+
+Stoplight Roles will also be mapped to External Roles. This mapping will be used when syncing permissions from external services to grant users the correct roles that match their roles in the external service.
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
Stoplight platform Roles are mapped to External Roles. These mappings are used when syncing permissions from external services to grant users the correct permissions that match their roles in the external service.
```

File: docs/external-services.md [hunk 6]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+
+We will provide the following default Stoplight role to external service role mappings for each external service kind we support.
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
We provide the following default Stoplight role to external service role mappings for each external service kind we support.
```

File: docs/external-services.md [hunk 7]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+
+*Bitbucket*
+
+Bitbucket Role | Stoplight Role
+---------------|---------------
+admin          | owner
+writer         | writer
+reader         | reader
+
+As with Roles and Role to permission mappings, Stoplight users will be able to edit these mappings in a future phase. We will start out with users not being able to edit these mappings.
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
As with Roles and Role to permission mappings, Stoplight users can edit these mappings in a future phase. We begin by users not being able to edit these mappings.
```

File: docs/external-services.md [hunk 8]
```diff
@@ -225,6 +225,78 @@ Connect External Service | Yes | No | This can occur at registration, or after r
...
+
+**Role/Permision Table Structure**
```
Comment (brianmrock) [inline docs/external-services.md]:
```suggestion
**Role/Permission Table Structure**
```

**Pull request merged**